### PR TITLE
multi sig migration files for root and contract registry  upgrades 

### PR DESCRIPTION
**Pull request by jaycenhorton**

### CODE REVIEW

File: test/helpers/contracts.js [hunk 1]
```diff
@@ -224,7 +225,138 @@ const upgradeToContract = async (
...
 };
 
+const deployOrGetProxy = async (
+  config,
+  existingProxyAddr,
+  contractRegistry,
+  multiAdmin,
+  deployParams
+) => {
+  const { artifacts } = config;
```
Comment (pcardune) [inline test/helpers/contracts.js]: Whenever I see destructuring of arguments as the first line in a function, I move the destructing to the function parameter declaration.

Comment (pcardune) [reply test/helpers/contracts.js]: In fact, whenever I have functions that take this many parameters, I use object destructuring for all of them so it's more obvious what's what at the function call sites.

Comment (jaycenhorton) [reply test/helpers/contracts.js]: I dont folllow the first comment

Comment (pcardune) [reply test/helpers/contracts.js]: Which is better:

```js
function log(config) {
  const {message, level, error, stack} = config;
  ...
}
```

or

```js
function log({message, level, error, stack}) {
  ...
}
```

My comment is that the second is always preferable to the first.

Comment (jaycenhorton) [reply test/helpers/contracts.js]: ah yes, thats clear. will modify on a future pr

**Pull request merged**